projects/ids-enterprise-ng/src/lib/swaplist/soho-swaplist.component.ts

```diff
@@ -148,6 +179,18 @@ export class SohoSwapListComponent implements AfterViewInit, OnDestroy {
   @Input()
   public fullAccessCardTitle = Soho.Locale.translate('AdditionalItems');
 
+  /** Default search title for available items card. */
+  @Input()
```

Comment:
Potentially unnecessary locale look ups?  Maybe add a getter which return the default if non provided?

Reply:
We made a translate pipe in landmark that will take 'AdditionalTimes | translate' in the template and run the pipe class against that. Apparently it's supposed to be more efficient too. Perhaps this should be in ep-ng?

```
import { Pipe, PipeTransform } from '@angular/core';

@Pipe({
  name: 'translate'
})
export class TranslatePipe implements PipeTransform {
  transform(value: string): string {
    return Soho.Locale.translate(value);
  }
}
```

Reply:
This is baked into `ids-enterprise-ng` as sohoTranslate.  

```html
 <div style="text-align: center;">
  <button soho-button="primary">{{'Yes'|sohoTranslate}}</button>
  <button soho-button="secondary">{{'No'|sohoTranslate}}</button>
</div>
```

There are also date and number formatters.

```html
 <div class="field">
   <input soho-textarea value="{{model.number|sohoFormatNumber}}"/>
   <input soho-textarea value="{{model.number|sohoFormatNumber:{style: 'currency'} }}"/>
  <input soho-textarea  value="{{model.number|sohoFormatNumber:{style: 'percent'} }}"/>
</div>
```

Reply:
I tried to remove all the `Soho.Locale.translate` and added the `{{'AdditionalItems'|sohoTranslate}}` to the label and input field but it seems the `sohoTranslate` couldn't find. I got an error after executing the `npm run build:lib`

Reply:
You may need to import the Locale pipe into the module.

Reply:
Still getting the error.